themes/classic/_dev/css/components/checkout.scss

```diff
@@ -222,6 +222,9 @@ body#checkout {
       label {
         text-align: inherit;
       }
+      img{
```

Comment:
Can you add a space before `{` please ? :smile:

Reply:
It's done